control-plane-operator/hostedclusterconfigoperator/controllers/inplaceupgrader/inplaceupgrader.go

```diff
@@ -211,7 +211,7 @@ func (r *Reconciler) reconcileInPlaceUpgrade(ctx context.Context, nodePoolUpgrad
 	// Signal in-place upgrade progress.
 	result, err := r.CreateOrUpdate(ctx, r.client, machineSet, func() error {
 		delete(machineSet.Annotations, nodePoolAnnotationUpgradeInProgressFalse)
-		machineSet.Annotations[nodePoolAnnotationUpgradeInProgressTrue] = fmt.Sprintf("Updating version in progress. Target version: %q. Total Nodes: %d. Upgraded: %d", *machineSet.Spec.Template.Spec.Version, len(nodes), len(nodes)-nodeNeedUpgradeCount)
+		machineSet.Annotations[nodePoolAnnotationUpgradeInProgressTrue] = fmt.Sprintf("Nodepool update in progress. Target Config hash: %s. Total Nodes: %d. Upgraded: %d", nodePoolUpgradeAPI.spec.targetConfigVersion, len(nodes), len(nodes)-nodeNeedUpgradeCount)
```

Comment:
why this change? Also you mention config hash but you show ConfigVersion

Reply:
Because now, the message in the Annotation shows like a version upgrade even if it's just a Config one, so it's confusing for the user. 

In the other one, I will change from `config hash` to `config version`